lib/reporters/html/view-model.js

```diff
@@ -32,13 +32,16 @@ module.exports = class ViewModel {
 
     /**
      * @param {TestStateResult} result
+     * @param {ReporterOptions} reporterOptions
      */
-    addSuccess(result) {
-        this._addTestResult(result, {
-            success: true,
-            actualPath: lib.currentPath(result),
-            expectedPath: lib.referencePath(result)
-        });
+    addSuccess(result, reporterOptions) {
+        if (!reporterOptions.failedOnly) {
+            this._addTestResult(result, {
+                success: true,
+                actualPath: lib.currentPath(result),
+                expectedPath: lib.referencePath(result)
+            });
+        }
 
         this._counter.onPassed(result);
```

Comment:
I agree, that we should display them. I don't think, that it would anyone astray, since the anyone would have to pass the --html-failed-only option and should have some rough idea, what should it do.